apps/picture_renditions.py

```diff
@@ -25,8 +29,15 @@ def create(self, docs, **kwargs):
             media_type, content_type = mimetype.split('/')
             renditions = generate_renditions(orig_file, orig['media'], inserted, media_type,
                                              mimetype, rendition_spec, app.media.url_for_media)
+
             doc['renditions'] = renditions
             ids.append(item['_id'])
+
+            updates = {'renditions': item['renditions']}
+            try:
+                get_resource_service('archive').update(item['_id'], updates, item)
+            except Exception as ex:
```

Comment:
any more specific exception we could use here?

Reply:
No, I think.

Reply:
@petrjasek Any feedback here?

Reply:
no